src/main/java/de/tum/in/www1/artemis/domain/quiz/QuizPointStatistic.java

```diff
@@ -120,7 +118,7 @@ public void removeOldResult(Double score, Boolean rated) {
      * @param countChange the int-value, which will be added to the Counter and participants
      */
     private void changeStatisticBasedOnResult(double score, Boolean rated, int countChange) {
-        double points = roundScoreSpecifiedByCourseSettings(quiz.getOverallQuizPoints() * (score / 100), quiz.getCourseViaExerciseGroupOrCourseMember());
+        double points = Math.round(quiz.getOverallQuizPoints() * (score / 100));
```

Comment:
please add a comment why we do not use `roundScoreSpecifiedByCourseSettings` here

Reply:
I added the comments in 3f66b64 in order to make it clear why course dependent rounding is not used at these two spots.